consensus/src/dag/dag_handler.rs

```diff
@@ -92,7 +92,7 @@ impl NetworkHandler {
                 .map(|r| r.into()),
             DAGMessage::CertifiedNodeMsg(node) => node
                 .verify(&self.epoch_state.verifier)
-                .and_then(|_| self.dag_driver.process(node))
+                .and_then(|_| self.dag_driver.process(node.certified_node()))
```

Comment:
It is a bit confusing that the nod has certified_node that has a node...
Maybe change the name of the variable?